### ISSUE TITLE
LEDText scrolls text on the visible screen if screen is higher than the font

### ISSUE THREAD

**Issue by marcmerlin** [AaronLiddiment/LEDText#30]
This was a problem with neopixels:
https://photos.app.goo.gl/BC24c4D6h68U92Rm8

and even more so with a bigger TFT display:
https://www.youtube.com/watch?v=PMSQ8xn8EVg

**Comment by AaronLiddiment** [comment]
The example4 you are using was made specifically for a 32x16 matrix with a 16 pixel high font. You will notice the extra spaces that have been added into the text message to make sure the text is where I wanted it before the change in direction took effect.
Also `Options ^= INSTANT_OPTIONS_MODE;`, this line makes the changes take place immeadiately rather than allowing the current text to scroll off the display before the change in direction.

**Comment by marcmerlin** [comment]
Right, I know my display is the wrong size, but is the text supposed to scroll up, hoping to be off display, but in my case it's on display and doesn't disappear?  (I understand what you're saying about left and right)

**Comment by AaronLiddiment** [comment]
Please read the Wiki on the `INSTANT_OPTIONS_MODE`
It explains that with this option set it will change text scroll direction instantly rather than waiting for the text to be off the display before changing direction. Comment out those lines and you will see the change it makes to the Example4 demo.

**Comment by CobaltEcho** [comment]
Ohhhh, so that's what instant options mode does!

**Comment by marcmerlin** [comment]
I commented out the 2 copies of INSTANT_OPTIONS_MODE, and I see the difference, but the scroll up still scrolls onto the displays. I understand that for you it was not an issue since your display wasn't higher than the font size.
This is not a big deal, I'm only going to be using scroll left and right anyway.
Feel free to close this issue, this is likely working as intended, scrolling only works as long as the scroll is towards the end of the display

**Comment by AaronLiddiment** [comment]
ok. I have noticed in your sample code I have seen that you set the text window to only be the font height + 1, you should set the text window to the full matrix height.
